### ISSUE TITLE
Pip instructions are incorrect

### ISSUE THREAD

**Issue by morgannunan** [ideasman42/nerd-dictation#126]
The instructions to [install this via pip](https://github.com/ideasman42/nerd-dictation/blob/main/package/python/readme.rst) just say `pip install package/python`.

**Comment by ideasman42** [comment]
What is the error you're getting?

**Comment by morgannunan** [comment]
It's just not clear to me how this would install nerd-dictation. Am I supposed to first install vosk with pip, then clone nerd-dictation with git, then cd into it, get the language model, and then run `pip install package/python` to finish installing nerd-dictation?

**Comment by ideasman42** [comment]
nerd-dictation is a stand alone script, there is no need to install it, you may symlink the script to `~/.local/bin` or some other location in your `$PATH`, I'd still be interested to know the errors you rain into.

**Comment by Moini** [comment]
Works for me - you need to run the given command from the nerddictation root folder.

@ideasman Installation is necessary for elograf. Simple symlinking probably will not take the modules with it, so unless you want to copy the whole git repo into the bin folder... it makes more sense to install.

**Comment by morgannunan** [comment]
I get it. I have to cd into the installl folder, then run `pip install package/python` and it will install whatever package is present in the current folder. This was the part that didn't make sense to me. Maybe that's ignorant of me, but you might consider editing these instructions, like so:

> To install nerd-dictaiton as a package from pip, run `pip install package/python` from within the nerd-dictation directory.

**Comment by Moini** [comment]
Yes. package/python is a path, not some kind of magic command (which I thought at first, too :) ).

**Comment by Free-Radical** [comment]
with all due respect this should be made much more clearer in the instructions

**Comment by ideasman42** [comment]
@morgannunan committed 03ce043a6d569a5bb9a715be6a8e45d8ba0930fd, does this resolve the issue for you?

**Comment by ideasman42** [comment]
Closing as the commit addresses the confusion.
